packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -109,6 +108,7 @@ export const Thumbnail: Comp<ThumbnailProps> = forwardRef((props, ref) => {
         size,
         theme,
         variant,
+        linkProps,
```

Comment:
Should be documented in the props interface along with the `linkAs` prop (that should be outside the `linkProps`)